installer/PowerToysSetup/Product.wxs

```diff
@@ -356,6 +341,25 @@
         <?endforeach?>
       </Component>
     </DirectoryRef>
+
+    <DirectoryRef Id="ApplicationProgramsFolder">
+      <Component Id="PowerToysStartMenuShortcut" Guid="336AB4F9-078C-4DCA-B69F-3808A9FFD758">
+        <Shortcut Id="ApplicationStartMenuShortcut"
+          Name="PowerToys (Preview)"
+          Description="PowerToys - Windows system utilities to maximize productivity"
+          Icon="powertoys.exe"
+          IconIndex="0"
+          Target="[!PowerToys.exe]"
+          WorkingDirectory="INSTALLFOLDER">
+          <ShortcutProperty Key="System.AppUserModel.ID" Value="Microsoft.PowerToysWin32"/>
+          <!-- ToastActivatorCLSID is used only by toast background activation, which currently isn't used, but causes MSI warning 1946 to display for a small share of users -->
+          <!-- <ShortcutProperty Key="System.AppUserModel.ToastActivatorCLSID" Value="{DD5CACDA-7C2E-4997-A62A-04A597B58F76}"/> -->
```

Comment:
Since it's not used, is there any reason to leave it commented out instead of removing it?

Reply:
In case we want to support this stuff in the future, otherwise it can be removed.

Reply:
Let's remove it.